metadata-ingestion/docs/transformer/dataset_transformer.md

```diff
@@ -175,6 +175,34 @@ transformers:
 The main use case of `simple_remove_dataset_ownership` is to remove incorrect owners present in the source. You can use it along with the [Simple Add Dataset ownership](#simple-add-dataset-ownership) to remove wrong owners and add the correct ones.
 
 Note that whatever owners you send via `simple_remove_dataset_ownership` will overwrite the owners present in the UI.
+## Extract Dataset globalTags
+### Config Details
+| Field                       | Required | Type         | Default       | Description                                                      |
+|-----------------------------|----------|--------------|---------------|------------------------------------------------------------------|
+| `extract_tags_from`         | ✅       | string       |  `urn`             | Which field to extract tag from. Currently only `urn` is supported.  |
+| `extract_tags_regex`        | ✅       | string       |  `.*`             | Regex to use to extract tag.|
+| `replace_existing`          |          | boolean      | `false`       | Whether to remove owners from entity sent by ingestion source.   |
+| `semantics`                 |          | enum         | `OVERWRITE`   | Whether to OVERWRITE or PATCH the entity present on DataHub GMS. |
+
+Let’s suppose we’d like to add a dataset tags based on part of urn. To do so, we can use the `extract_dataset_tags` transformer that’s included in the ingestion framework.
```

Comment:
to clarify - this can only add a single tag right?
```suggestion
Let’s suppose we’d like to add a dataset tag based on part of urn. To do so, we can use the `extract_dataset_tags` transformer that’s included in the ingestion framework.
```